docker-compose.yaml

```diff
@@ -3,6 +3,32 @@ version: '3.4'
 services:
     db:
         image: postgres:11.2-alpine
+    s3:
+        image: minio/minio:RELEASE.2019-05-02T19-07-09Z
+        environment:
+            MINIO_ACCESS_KEY: &S3Key AKIAIOSFODNN7EXAMPLE
+            MINIO_SECRET_KEY: &S3Secret wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
+            MINIO_REGION: &S3Region us-east-1
+        command: server /app
+        ports:
+            - 8081:9000
+    s3-bucket:
```

Comment:
is this the abstraction layer?

Reply:
`minio/mc` is the command-line tool for `minio/minio`. Not ideal having it, but had some access problems without.

Reply:
Looked at options, no easy alternative that I can see.

Reply:
re-reading, this is just an init container (as opposed to a service) for the `s3` service, should work well. Assumption is both of them are not needed in a context where a real S3 bucket is available.